glusterd2/volume/struct.go

```diff
@@ -35,6 +35,16 @@ const (
 const (
 	// VkeyFeaturesQuota is the key to check if quota is disabled or enabled
 	VkeyFeaturesQuota = "features.quota"
+	// VkeyQuotaDeemStatfs is the key to disabled or enabled deem statfs
```

Comment:
These shouldn't be in `glusterd2/volume/struct.go` as these are xlator (quota) specific. I guess it can be moved during refactor later.

Reply:
If this is only for volume options where do I put the quota options. 
I searched for the right place to put it couldn't find it. 
I can see that bitrot has too put it here. 
Do I have to use the string as such everywhere or is there a place to put it which I'm not able to find?

Reply:
> I guess it can be moved during refactor later.

We'll move it later. For now, let it be.

Reply:
On second thought, you can move these definitions to `./plugins/quota/consts.go`. In future, we'll find a better place for these validation functions itself.

Reply:
done